src/Models/Note.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Tipoff\Notes\Models;
```

Comment:
There is a typo here, Models should be before Notes in the namespace.

Reply:
@drewroberts Can you explain this to me? The composer.json says the package namespace is Tipoff\Notes so shouldn't the namespace be Tipoff\Notes and then the name of the directory in this case Models?

Reply:
Haha, the typo was in my head. You are correct. My bad!